content/docs/atom-guide/atom-sample-viewer/debug-menu-samples.md

```diff
@@ -11,11 +11,54 @@ This section dives into the debug menu samples demonstrated in the Atom Sample V
+
+* Swap to statistics is used to swap between the statistics view and the visualizer.
+* Resume can be used to start or stop profiling data collection from Atom.
+* Capture takes a single-frame capture of CPU profiling data and saves it to a location on disk, by default into /<project>/user/CpuProfiler/.
+* Begin can be used to start a continuous capture of CPU profiling data, during which Atom is storing many frames of data. End will end the continuous capture and save the data that was collected to disk, by default into /<project>/user/CpuProfiler/.
+* Load File will bring up the file picker window which will list all of the.json files in /<project>/user/CpuProfiler/. This list of files is sorted in decreasing order according to modification time, so the most recent capture is at the top.
+
+### Loading Captures
+
+![CPU Profiler File Picker](/images/atom-guide/atom-sample-viewer/cpu-profiler-file-picker.png)
+
+* Load Selected will load the selected capture into memory, after which the data will be visible in the statistics and visualizer views.
+
+### Statistics
+
+![CPU Profiler Statistics](/images/atom-guide/atom-sample-viewer/cpu-profiler-statistics.png)
+
+The statistics view focuses on processing and displaying the quantitative data collected by Atom's CPU profiler. Each row in the table represents a specific profiling region, and all columns in the table can be sorted by clicking on the column headers:
+* Group: The overall renderer area (ex. RPI, RHI, DX12) that the profiling region occurs in.
+* Region: The lower-level name of the profiling region, often the name of a function call.
+* MTPC: The mean time per call in milliseconds of the profiling region aggregated across all of the data collected while the CPU profiler widget has been active.
+* Max: The maximum execution time of the region in milliseconds during the last frame's execution. For regions that only appeared once in the last frame, this will be equal to the total time.
+* Invocations: The number of times that the profiling region was executed in the last frame.
+* Total: The overall time spent in milliseconds executing this profiling region on the last frame, summing up each invocation of the region.
+
+Rows in the table with a specific group or region name can also be searched for with the filter, which can be easily cleared by clicking  Clear Filter. Reset Table will clear all of the table's saved data and is recommended when switching samples.
+
+### Visualizer
+
+![CPU Profiler Statistics](/images/atom-guide/atom-sample-viewer/cpu-profiler-visualizer.png)
+
+The visualizer presents the CPU profiling data in a timeline format similar to other profilers like RAD Telemetry and Tracy. Each active render thread is laid out horizontally, with every profiling event corresponding to a block in the visualizer. Time increases going to the right.
+
+To scroll throughout the data, hold **RMB+Drag**. To zoom in and out, use **Control+Mouse Wheel**.
```

Comment:
**Option A:**
I suggest reformatting this into an "**action** : description" list to make it easier to read and reference. 
```suggestion
You can navigate through the timeline using the following controls:
- **RMB+Drag**: To scroll through the data.
- **Ctrl+Mouse Wheel**: To zoom in and out.
```

Reply:
**Option B**: 
Combine these interactions with the other set of interactions, such as using the **Saved Frames** slider or **Find Region** text field. (See the last **Option B** suggestion.)
```suggestion
```

Reply:
First one seems easier to follow